web/modules/custom/asu_statistics/asu_statistics.module

```diff
@@ -0,0 +1,149 @@
+<?php
+
+/**
+ * @file
+ * Contains asu_statistics.module.
+ */
+
+/**
+ * Implements hook_theme().
+ */
+function asu_statistics_theme($existing, $type, $theme, $path) {
+  return [
+    'asu_statistics_chart' => [
+      'variables' => [
+        'form' => NULL,
+        'download_url' => NULL,
+        'download_stat_summary_url' => NULL,
+        'total_items' => NULL,
+        'stats_table' => NULL,
+        'summary_row' => NULL,
+        'content_counts_table' => NULL,
+        'total_collections' => NULL,
+        'collections_by_institution' => NULL,
+      ],
+    ],
+  ];
+}
+
+/**
```

Comment:
does this function work since it isn't prefaced with asu_statistics?

Reply:
I am not 100% sure - it is only a harmless bit that could be in the twig itself yet I copied and removed a lot of stuff that was previously happening in islandora_repository_reports ... his was named similarly:
https://github.com/mjordan/islandora_repository_reports/blob/main/islandora_repository_reports.module#L42-L80

Reply:
ah ok - must be this one: 
>template_preprocess_HOOK(&$variables): Should be implemented by the module that registers the theme hook, to set up default variables.

Reply:
so that function you linked to would be setting default variables for https://github.com/mjordan/islandora_repository_reports/blob/main/islandora_repository_reports.module#L21

Reply:
and in our case: settings defaults for https://github.com/asulibraries/islandora-repo/pull/249/files/65e5c606dbf3c886cbd6378f0772891498c1c939#diff-5a56af7bbf7a1720d2c37547cff2fdddda3f984dabda4b4bf91412cde0cfa70cR13